### DISCUSSION TITLE
Serilog.Sinks.Http 8.0.0 does not seem to work with Serilog.AspNetCore 6.0.1

### DISCUSSION THREAD

**Discussion by hoaol** [FantasticFiasco/serilog-sinks-http#266] [Q&A]
Hi! I have a .NET 6 ASP application where I have updated the `Serilog.AspNetCore` package to 6.0.1 and at the same time also upgraded `Serilog.Sinks.Http` to 8.0.0. After that the logging to elastic stopped working (we can still log to file and console). If I revert to 7.2.0 it works again. We are configuring the sink via appsettings.json:
```
{
                "MinimumLevel": "Debug",
                "Name": "Http",
                "Args": {
                    "requestUri": "http://myElasticServer:9606/",
                    "queueLimitBytes": null
                }
}
```
I have checked the list of breaking changes but do not anything applies to our usage.
Does anyone have an idea what could cause this problem? Right now it is no problem for me to keep using 7.2.0 but in the long term I would like to "upgrade".

Kind regards Hans

**Comment by FantasticFiasco** [comment]
Hi @hoaol!

It's currently summer vacations here in Sweden now. I'll look at your issue when I return.

**Comment by FantasticFiasco** [comment]
Hi @hoaol. Is this still a problem?

**Reply by hoaol** [reply]
We can use the 7.2.0 version for now and if nobody else ha reported anything similar to this I suppose we can wait until we need to step up to a 8.* version and maybe do some more investigation. I suppose the problem could be in `Serilog.AspNetCore` as well (or in our application)
Br Hans

**Comment by hoaol** [comment]
We have now encountered the same problem in another project, would it be easy for you to verify that  `Serilog.Sinks.Http 8.0.0` works with `Serilog.AspNetCore 6.0.1` (maybe that has already been done)? 

Do you think that there were any changes made in 8.0.0 that would require us to change how we use it?

**Comment by hoaol** [comment]
I just wanted to mention that I have now resolved this, I had missed that the format for "batched" log events had changes (there is no longer an `events` property with the log items) and we had a logstash "transformation" on the logs (that I had fogotten about) that depended on the old "batch" format 

    Version 8.0.0 or later
```
    /// <summary>
    /// Formatter serializing batches of log events into a JSON array.
    /// <para/>
    /// Example:
    /// [
    ///   { event n },
    ///   { event n+1 }
    /// ]
    /// </summary>
    public class ArrayBatchFormatter : IBatchFormatter
```


    Version 7.2.0
```
    /// <summary>
    /// Formatter serializing batches of log events into a JSON object under a property called "events".
    /// <para/>
    /// Example:
    /// {
    ///   "events": [
    ///     { event n },
    ///     { event n+1 }
    ///   ]
    /// }
    /// </summary>
    public class DefaultBatchFormatter : BatchFormatter
```
Br Hans

**Reply by FantasticFiasco** [reply]
Thanks for letting me know!
